client/my-sites/plugins/plugin-meta/index.jsx

```diff
@@ -88,7 +88,7 @@ const PluginMeta = React.createClass( {
 	},
 
 	isWpcomPreinstalled: function() {
-		const installedPlugins = [ 'Jetpack by WordPress.com', 'Akismet' ];
+		const installedPlugins = [ 'Jetpack by WordPress.com', 'Akismet', 'VaultPress' ];
```

Comment:
Not directly related to this issue, but I've noticed that we are not showing VaultPress as already installed.